File: DoneItAPI/DoneItAPI/DTOs/TareaRequest.cs
namespace DoneItAPI.DTOs
{
    public class TareaRequest
    {
        public string Titulo { get; set; }
        public string? Descripcion { get; set; }
        public DateTime? Fecha_Inicio { get; set; }
        public DateTime? Fecha_Fin { get; set; }
        public string Estado { get; set; }
        public string Prioridad { get; set; }
        public int Id_Proyecto { get; set; }
    }
}

File: DoneIt_Version_Movil/app/src/main/java/com/herreroco/doneit_version_movil/model/TareaRequest.kt
package com.herreroco.doneit_version_movil.model

data class TareaRequest(
    val titulo: String,
    val descripcion: String?,
    val fecha_Inicio: String?, // <- esto
    val fecha_Fin: String?,    // <- esto
    val estado: String,
    val prioridad: String,
    val id_Proyecto: Int
)
